client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,195 @@
+					<FormFieldset>
+						<ReduxFormToggle
+							name="job_manager_user_requires_account"
+							text={ translate( 'Submitting listings requires an account' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Non-logged in users will be unable to submit listings without creating an account.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_enable_registration"
+							text={ translate( 'Allow account creation' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Non-logged in users will be able to create an account by entering ' +
+								'their email address on the submission form.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_generate_username_from_email"
+							text="Automatically generate username from email address" />
+						<FormSettingExplanation isIndented>
+							{ translate( 'A username will be generated from the first part of the ' +
+								'user email address. Otherwise, a username field will be shown.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Role' ) }
+						</FormLabel>
+						<ReduxFormSelect name="job_manager_registration_role">
+							<option value="editor">{ translate( 'Editor' ) }</option>
+							<option value="author">{ translate( 'Author' ) }</option>
+							<option value="contributor">{ translate( 'Contributor' ) }</option>
+							<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+							<option value="teacher">{ translate( 'Teacher' ) }</option>
+							<option value="employer">{ translate( 'Employer' ) }</option>
+							<option value="customer">{ translate( 'Customer' ) }</option>
+							<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+						</ReduxFormSelect>
+						<FormSettingExplanation>
+							{ translate( 'If you enable registration on your submission form, choose a role for the new user.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</form>
+			</Card>
+
+			<SectionHeader label={ translate( 'Approval' ) }>
+				<FormButton compact primary />
+			</SectionHeader>
+			<Card>
+				<form>
+					<FormFieldset>
+						<ReduxFormToggle
+							name="job_manager_submission_requires_approval"
+							text={ translate( 'New listing submissions require admin approval' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'New submissions will be inactive, pending admin approval.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_user_can_edit_pending_submissions"
+							text="Submissions awaiting approval can be edited" />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Submissions awaiting admin approval can be edited by the user.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</form>
+			</Card>
+
+			<SectionHeader label={ translate( 'Listing Duration' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Duration' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).